docs_src/design/adr/security/0026-authentication.md

```diff
@@ -0,0 +1,296 @@
+# EdgeX Microservice Authentication
```

Comment:
Are we going to provide a UCR doc for this need or straight to ADR?

Reply:
@jpwhitemn #898 has the UCR.